FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,16 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(RemoteFeedImageMapper.map(data, from: response))
+			case .failure(_):
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
`RemoteFeedLoader.` is not needed since this line is inside the scope, please remove `RemoteFeedLoader.`

```suggestion
				completion(.failure(Error.connectivity))
```